src/macaron/repo_finder/repo_finder_enums.py

```diff
@@ -0,0 +1,116 @@
+
+    #: Reported if the URL of the repository could not be reached.
+    HTTP_INVALID = "HTTP invalid"
+
+    #: Reported if the URL host returns a 404 (not found) for the sought resource.
+    HTTP_NOT_FOUND = "HTTP not found"
+
+    #: Reported if the URL host returns a 403 (forbidden) for the sought resource.
+    HTTP_FORBIDDEN = "HTTP forbidden"
+
+    #: Reported for all other bad status codes that a host could return. E.g. 500, etc.
+    HTTP_OTHER = "HTTP other"
+
+    #: Reported if deps.dev produces no response to the HTTP request.
+    DDEV_BAD_RESPONSE = "deps.dev bad response"
+
+    #: Reported if deps.dev returns JSON data that cannot be parsed.
+    DDEV_JSON_FETCH_ERROR = "deps.dev fetch error"
+
+    #: Reported if deps.dev returns JSON data that is missing expected fields.
+    DDEV_JSON_INVALID = "deps.dev JSON invalid"
+
+    #: Reported if deps.dev returns data that does not contain the desired SCM URL. E.g. The repository URL.
+    DDEV_NO_URLS = "deps.dev no URLs"
+
+    #: Reported if the provided PURL did not produce a result, but a more recent version could not be found.
+    NO_NEWER_VERSION = "No newer version than provided which failed"
+
+    #: Reported if the provided PURL did not produce a result, and the most recent version also failed.
+    LATEST_VERSION_INVALID = "A newer version than provided also failed"
+
+    #: Reported when a repository is found.
+    FOUND = "Found"
+
+    #: Reported when a repository is found via a parent POM.
+    FOUND_FROM_PARENT = "Found from parent"
+
+    #: Reported when a repository is found from a more recent version than was provided by the user.
+    FOUND_FROM_LATEST = "Found form latest"
+
+    #: Default value. Reported if the Repo Finder was not called. E.g. Because the repository URL was already present.
+    NOT_USED = "Not used"
+
+
+class CommitFinderOutcome(Enum):
+    """An Enum of all outcomes of the Commit Finder being run for a software component."""
+
+    #: Reported if the user does not provide a version for the Repo Finder via the command line, or allow the version to
+    # be automatically discovered.
+    NO_VERSION_PROVIDED = "No version provided"
+
+    #: Reported if the user provides an unsupported type in the PURL command line argument.
+    UNSUPPORTED_PURL_TYPE = "Unsupported PURL type"
+
+    #: Reported if the user provided a repository type PURL with the tag or commit in the version, but neither were
+    # valid.
+    REPO_PURL_FAILURE = "Repository PURL failure"
+
+    #: Reported if the repository has no tags.
+    NO_TAGS = "No tags"
+
+    #: Reported if the repository has no tags with associated commits.
+    NO_TAGS_WITH_COMMITS = "No tags with commits"
+
+    #: Reported if the tag selected from the repository fails to resolve to a commit despite having one associated with
+    # it.
+    NO_TAG_COMMIT = "No tag commit"
+
+    #: Reported if the version part of the user provided PURL is invalid.
```

Comment:
The comment only says the `version` part. Is that intended?

Reply:
Yes. I've changed it to `INVALID_VERSION`